.env.example

```
@@ -68,6 +68,11 @@ AXON_OPENAI_API_KEY=
 AXON_CODEX_COMPLETION_CONCURRENCY=
 AXON_CODEX_MODEL=
 AXON_SYNTHESIS_CODEX_MODEL=
+# Opt-in escape hatch: when true, run codex app-server against the user's REAL
+# CODEX_HOME with the full inherited environment, so MCP servers, skills, and
+# hooks load. This surrenders the synthesis isolation above — leave false unless
+# you specifically want tool-enabled (agentic) Codex behavior.
+AXON_CODEX_LOAD_USER_CONFIG=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AXON_CODEX_LOAD_USER_CONFIG key should go before the AXON_CODEX_MODEL key

(UnorderedKey)